src/View/Helper/UrlHelper.php

```diff
@@ -218,9 +218,11 @@ public function assetTimestamp($path)
             if (Plugin::loaded($plugin)) {
                 unset($segments[0]);
                 $pluginPath = Plugin::path($plugin) . 'webroot' . DIRECTORY_SEPARATOR . implode(DIRECTORY_SEPARATOR, $segments);
-                //@codingStandardsIgnoreStart
-                return $path . '?' . @filemtime($pluginPath);
-                //@codingStandardsIgnoreEnd
+                if (file_exists($pluginPath)) {
+                    //@codingStandardsIgnoreStart
+                    return $path . '?' . @filemtime($pluginPath);
```

Comment:
If the file exists, you don't need to suppress errors when getting its modified time.